pydbtools/_wrangler.py

```diff
@@ -304,6 +313,80 @@ def create_temp_table(
     ath.wait_query(q_e_id, boto3_session=boto3_session)
 
 
+def _create_temp_table_in_sql(sql: str) -> bool:
+    """
+    Allows the user to write SQL of the format
+    CREATE TEMP TABLE tablename AS (...)
+
+    Args:
+        sql (str):
+            An SQL query.
+
+    Returns:
+        A bool indicating whether a temporary table was
+        created (True) or whether the SQL still needs to
+        be processed (False).
+    """
+
+    sql = clean_query(sql, fmt_opts={"strip_comments": True}).lower()
```

Comment:
Does this not alter the SQL in ways we wouldn't want e.g.:

```sql
select * from db.table where answer = 'Y';
--- Would get changed to
select * from db.table where answer = 'y';
```

Reply:
Yes, good point.

Reply:
Fixed - https://github.com/moj-analytical-services/pydbtools/pull/60/commits/6ed0207820c2d0c45a3eb093e3412dffeee93b3a